app/services/submissions_sort_and_filter_service.rb

```diff
@@ -67,10 +68,24 @@ def apply_sorting
       @submissions = @submissions.order_by_average_score(:desc)
     when 'average_score_low_to_high'
       @submissions = @submissions.order_by_average_score(:asc)
+    when 'assignees_high_to_low'
+      @submissions = @submissions.order_by_assignee_count(:desc)
+    when 'assignees_low_to_high'
+      @submissions = @submissions.order_by_assignee_count(:asc)
     when 'submission_id_high_to_low'
       @submissions = @submissions.order(id: :desc)
     when 'submission_id_low_to_high'
       @submissions = @submissions.order(id: :asc)
     end
   end
+
+  def apply_includes
+    association_preloads = if @params[:sort]&.include?('assignees')
+                             { evaluator_submission_assignments: [:evaluator, { evaluation: :evaluation_scores }] }
+                           else
+                             { evaluator_submission_assignments: [:evaluator, :evaluation] }
+                           end
+
+    @submissions = @submissions.includes(association_preloads)
```

Comment:
there's an rspec failure and the sort by assignee views have 500 internal server error but the simple fix is to use preload instead of includes. 
also getting a bullet warning on the other views because of excluding the `evaluation_scores` from the query.
<img width="704" alt="Screenshot 2025-02-25 at 5 02 12 PM" src="https://github.com/user-attachments/assets/672f6f82-bf05-4be5-988a-280466d8ad76" />
this works for both:
```suggestion
    association_preloads = { evaluator_submission_assignments: [:evaluator, { evaluation: :evaluation_scores }] }
    @submissions = @submissions.preload(association_preloads)
```